src/main/resources/edu/cornell/kfs/fp/document/datadictionary/RecurringDisbursementVoucherDocument.xml

```diff
@@ -0,0 +1,130 @@
+				<ref bean="DisbursementVoucherDocument-disbVchrContactEmailId" />
+				<ref bean="DisbursementVoucherDocument-disbursementVoucherDueDate" />
+				<ref bean="DisbursementVoucherDocument-disbVchrAttachmentCode" />
+				<ref bean="DisbursementVoucherDocument-disbVchrSpecialHandlingCode" />
+				<ref bean="DisbursementVoucherDocument-disbVchrCheckTotalAmount" />
+				<ref bean="DisbursementVoucherDocument-disbursementVoucherDocumentationLocationCode" />
+				<ref bean="DisbursementVoucherDocument-dvCheckStubOverflowCode" />
+				<ref bean="DisbursementVoucherDocument-disbVchrPayeeTaxControlCode" />
+				<ref bean="DisbursementVoucherDocument-disbVchrPayeeChangedInd" />
+				<ref bean="DisbursementVoucherDocument-disbursementVoucherCheckNbr" />
+				<ref bean="DisbursementVoucherDocument-disbursementVoucherCheckDate" />
+				<ref bean="DisbursementVoucherDocument-disbVchrPayeeW9CompleteCode" />
+				<ref bean="RecurringDisbursementVoucherDocument-disbVchrPaymentMethodCode" />
+				<ref bean="DisbursementVoucherDocument-campusCode" />
+				<ref bean="DisbursementVoucherDocument-disbVchrCheckStubText" />
+				<ref bean="DisbursementVoucherDocument-extractDate" />
+				<ref bean="DisbursementVoucherDocument-paidDate" />
+				<ref bean="DisbursementVoucherDocument-cancelDate" />
+				<ref bean="DisbursementVoucherDocument-disbursementVoucherPdpStatus" />
+				<ref bean="DisbursementVoucherDocument-disbVchrBankCode" />
+				<ref bean="DisbursementVoucherDocument-disbVchrPdpBankCode" />
+				<ref bean="DisbursementVoucherDocument-disbExcptAttachedIndicator" />
+				<ref bean="DisbursementVoucherDocument-immediatePaymentIndicator" />
+				<ref bean="RecurringDisbursementVoucherDocument-paymentCancelReason" />
+			</list>
+		</property>
+	</bean>
+	
+	<bean id="RecurringDisbursementVoucherDocument-disbVchrPaymentMethodCode" parent="RecurringDisbursementVoucherDocument-disbVchrPaymentMethodCode-parentBean" />
+	<bean id="RecurringDisbursementVoucherDocument-disbVchrPaymentMethodCode-parentBean" abstract="true" parent="DisbursementVoucherDocument-disbVchrPaymentMethodCode">
+		<property name="name" value="disbVchrPaymentMethodCode" />
+		<property name="control">
+			<bean parent="SelectControlDefinition"
+				p:valuesFinderClass="edu.cornell.kfs.fp.businessobject.options.RecurringDisbursementVoucherPaymentMethodValuesFinder"
+				p:includeKeyInLabel="false" />
+		</property>
+	</bean>
+	
+	<bean id="RecurringDisbursementVoucherDocument-paymentCancelReason" parent="RecurringDisbursementVoucherDocument-paymentCancelReason-parentBean" />
+	<bean id="RecurringDisbursementVoucherDocument-paymentCancelReason-parentBean" abstract="true" parent="AttributeDefinition">
+		<property name="name" value="paymentCancelReason"/>
+	    <property name="forceUppercase" value="false"/>
+	    <property name="label" value="Payment Cancel Reason"/>
+	    <property name="shortLabel" value="Payment Cancel Reason"/>
+	    <property name="maxLength" value="200"/>
```

Comment:
Four rows of 75 columns is 300; seems like there may be a mismatch between the control size and the maxLength?